pkg/actuators/awsplacementgroup/controller_test.go

```diff
@@ -1,58 +0,0 @@
-package awsplacementgroup
```

Comment:
Does this need to be deleted?

Reply:
Now that we reintroduced the types, we can bring this back.